.gitignore

```diff
@@ -27,3 +27,4 @@ appserver/extras/arquillian-containers/payara-common/dependency-reduced-pom.xml
 /nucleus/payara-modules/service-exemplar/nbproject/
 /nucleus/admin/server-mgmt/nbproject/
 appserver/tests/quicklook/quicklook_summary.txt
+/nucleus/admin/rest/rest-service/nbproject/
```

Comment:
Would you mind removing all lines similar to this and adding a general `nbproject/`? Adding specific directories is generally bad practice, as which is why there's a vast amount of redundancy in this file

Reply:
Would adding **/nbproject work? I think this got added automatically

Reply:
I don't believe it even needs the `**` at the start, although I would check just to be sure!